plugins/frontend/backstage-plugin-argo-cd/config.d.ts

```diff
@@ -6,5 +6,29 @@ export interface Config {
      * @visibility frontend
      */
     baseUrl?: string;
+    /**
+     * The base url of the ArgoCD instance.
+     * @visibility frontend
+     */
+    appLocatorMethods?: Array</**
+     * @visibility secret
+     */
+    {
+      /**
+       * The base url of the ArgoCD instance.
+       * @visibility frontend
+       */
+      type: string;
+      instances: Array<{
```

Comment:
Are these actually ever needed on the frontend? We only care that the `appLocatorMethods` exists and is not empty right? We could possibly omit the `@visibility frontend` from these subitems or omit them completely, leaving them only present in the backend `config.d.ts`. I'm not sure if it changes the boolean check logic but I feel it might be fine.

Reply:
addressed. seems to be working just fine. tests are passing and its working locally still.